lib/src/main/java/com/github/kpdlib/traditional/Surf.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.kpdlib.traditional
+
+import com.github.kpdlib.KeypointDetector
+
+/**
+ * Wrapper for a native
+ * [OpenCV SURF detector](https://docs.opencv.org/4.6.0/d5/df7/classcv_1_1xfeatures2d_1_1SURF.html).
+ *
+ * - Keypoints have sizes and angles
+ * - Descriptor size is 64
+ */
+class Surf(width: Int, height: Int) :
```

Comment:
Can all algos be sealed classes?

Reply:
Yes, they can, but I don't see why they should be. I don't see anything in the library that will break if a user would subclass an algorithm, so why restrict them to do so?